metrics-observer-prometheus/src/lib.rs

```diff
@@ -28,6 +35,23 @@ impl PrometheusBuilder {
         self.quantiles = parse_quantiles(quantiles);
         self
     }
+
+    /// Sets the buckets to use when rendering summaries.
+    ///
+    /// Buckets values represent the higher bound of each buckets.
```

Comment:
Can you add a sentence here (and to `set_buckets_for_metric` as well) that specifying buckets will change the default behavior from exporting a histogram and instead will export a summary for the given metric(s)?